### PR TITLE
Test all default options

### PR DESCRIPTION
**Pull request by r-ash**
This PR will
* Add a test that creates options JSON for both model and calibrate and validates them against schema

Note there is a change to the schema that I will have to update in hintr when it comes to integrate this (namely being able to have a default with 2 values selected instead of just 1)

This I think is the last piece required to be able to use this in hintr. I want to add 1 more function to map from an `id` to the `label` which will be used in naomi to get labels from the ID for the report

### CODE REVIEW

File: scripts/copy_schemas.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+import requests
```
Comment (richfitz) [inline scripts/copy_schemas.py]: what is wrong with curl here?

```
#!/usr/bin/env bash
set -e
curl https://raw.githubusercontent.com/mrc-ide/hintr/master/inst/schema/ModelRunOptions.schema.json > inst/schema/ModelRunOptions.schema.json
```

Comment (richfitz) [reply scripts/copy_schemas.py]: (also this is why I like scripts with no extension, as you're free to change the implementation without updating README files)

**Review by richfitz — COMMENTED** [review]
LGTM, just a small comment on the download script

File: scripts/copy_schemas.py (hunk 1 above)
Comment (r-ash) [reply scripts/copy_schemas.py]: No good reason! I started writing python expecting it to be more complicated than it was. Have  updated to much simpler script. Thanks

**Review by richfitz — APPROVED** [review]

**Pull request merged**